ProcessLib/ComponentTransport/ComponentTransportProcess.cpp

```diff
@@ -61,9 +61,19 @@ void ComponentTransportProcess::assembleConcreteProcess(
     DBUG("Assemble ComponentTransportProcess.");
 
     // Call global assembler for each local assembly item.
-    GlobalExecutor::executeMemberDereferenced(
-        _global_assembler, &VectorMatrixAssembler::assemble, _local_assemblers,
-        *_local_to_global_index_map, t, x, M, K, b, *_coupling_term.lock());
+    if (_coupling_term.lock())
+    {
+        GlobalExecutor::executeMemberDereferenced(
```

Comment:
I'm not sure if avoiding the `if (_coupling_term.lock())` inside the `VectorMatrixAssembler` is worth that much code duplication...